introduction.md

```diff
@@ -0,0 +1 @@
+Aloha mighty Dojo, I am Bill Murrin. It is a pleasure to meet you.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.